src/main/java/project/bot/model/dao/HUserDao.java

```diff
@@ -29,14 +30,15 @@ public User getByChatId(Long chatId) {
     @Override
     public void addUser(User user) {
         try (Session session = HibernateSessionFactoryUtil.getSessionFactory().openSession()) {
-            User existingUser = session.byNaturalId(User.class)
-                    .load();
+            User existingUser = session.bySimpleNaturalId(User.class).load(user.getChatId());
             if (existingUser == null) {
                 session.getTransaction().begin();
                 session.persist(user);
                 session.getTransaction().commit();
+                log.info(user.toString() + ", is added to DB");
```

Comment:
Its not secure to print user information into logs. Please add more common log message